.github/workflows/docker-env-release-api-tests.yml

```diff
@@ -101,4 +101,4 @@ jobs:
     with: 
       # TODO update environment name when there are more clusters
       environment_name: "dev"
-      service_name: ${{ inputs.working_dir }}
```

Comment:
Please also add a line:
`service_name: ${{ inputs.service_name }}`

Reply:
So Dave and I chatted and we omitted the service name since the api-workflow just needs the working directory.